_posts/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product.markdown

```diff
@@ -0,0 +1,120 @@
+
+**Use consistent terminology and workflows across all tools and features to help customers work faster and build intuitive mental models.** There are several instances where the experience changes depending on whether users are logged in or not. There are also instances where terms like “buckets” are used that differ from industry standards. Simplifying terminology and workflows to follow industry knowledge and best practices will increase customer confidence that this product is for them.
+
+**Minimize available actions within the Playground that trial users cannot access**. A recurring Tenet & Trap violation was Inviting Dead End. Users encounter options (e.g. “add sample data”) within the Playground that they do not have access to and are unable to perform. Focusing on eliminating these encounters and errors make it easier for potential customers to assess the platform.
+
+**Investigate potential content gaps on [OpenSearch.org](http://opensearch.org/) which** may hinder new user trial. The customer journey on [OpenSearch.org](http://opensearch.org/) seems to have information gaps that add friction to understanding and trialing of the Playground.
+
+
+## Takeaways
+
+* Perform a heuristics analysis to rapidly identify common traps and violations in your product design
+* You can scope your research as broadly or narrowly as you need
+* UX heuristics insights are often universally applicable to your entire UX; be sure to broaden your understanding (and dissemination) of your findings to ensure maximum value from your efforts
+* You can engage outside UX research experts or utilize existing resources to apply these UI Tenets and Traps and 10 Usability Heuristics to your product
+* Implement a regular Heuristics research practice to monitor and track improvements.
+
+## References
+
+[UI Tenets & Traps](https://uitraps.com/)
+[Nielsen’s 10 Usability Heuristics](https://www.nngroup.com/articles/ten-usability-heuristics/)
+
+
+### About Steyer
+
+
+"We don't just analyze data, we create meaning from it. We don't just report insights, we inspire action from them."
+
+Steyer partners with our clients to make meaningful connections between people and information, solving our clients’ business problems with the tools we know best: strategic analysis, user research, and the creation, organization, and revision of business content. With nearly three decades of delighting both our clients and our own team members, we understand what it takes to effect real change in the real world.
+
+![](/assets/media/blog-images/2024-07-19-using-heuristic-evaluation-to-inform-the-user-experience-strategy-of-a-product/Image1.jpg)
+
+
+**About Ray Salas - Steyer Insight Research Lead and Strategist**
```

Comment:
Deleted, as this information is already provided in rsalas.md. If the image is not already included in that file, please relocate.